src/components/WorkspaceContainer.js

```diff
@@ -170,175 +177,178 @@ function WorkspaceContainer() {
   })
 
   return (
-    <SelectionsContextProvider
-      selections={selections}
-      onSelections={setSelections}
-      quote={selectedQuote?.quote}
-      occurrence={selectedQuote?.occurrence}
-      verseObjects={originalScriptureConfig.verseObjects || []}
-    >
-      {loading || content || error ?
-        <DraggableCard
-          open
-          error={error}
-          title={title}
-          loading={loading}
-          content={content}
-          onClose={() => clearContent()}
-        />
-        :
-        null
-      }
-      <Workspace
-        rowHeight={25}
-        layout={layout}
-        classes={classes}
-        gridMargin={[15, 15]}
-        onLayoutChange={setCurrentLayout}
+    !workspaceReady ? // do not render workspace until user logged in and we have user settings
+      <CircularProgress size={180} />
+      :
+      <SelectionsContextProvider
+        selections={selections}
+        onSelections={setSelections}
+        quote={selectedQuote?.quote}
+        occurrence={selectedQuote?.occurrence}
+        verseObjects={originalScriptureConfig.verseObjects || []}
       >
-        <ScriptureCard
-          cardNum={0}
-          title='Literal Translation'
-          reference={{
-            chapter,
-            verse,
-            bookId,
-            projectId: bookId,
-          }}
-          resource={{
-            owner,
-            languageId,
-            resourceId: TARGET_LITERAL,
-            originalLanguageOwner: scriptureOwner,
-          }}
-          {...commonScriptureCardConfigs}
-        />
+        {loading || content || error ?
+          <DraggableCard
```

Comment:
Why is the draggable card use for?

Reply:
@mannycolon - github compare is a mess. you can see it on line 180 of the previous version.  Something you added 3/29.